src/helpers/node.ts

```diff
@@ -39,6 +39,15 @@ export async function resolveDependencyPath(packageName: string): Promise<string
   }
 }
 
+/**
+ * Get the atom home which includes `packages` folder.
+ * Test runners such as `atom-jasmine3-runner` change this folder in the test env,
+ * so we need to install packages in the correct path
+ */
+function getAtomHomePath() {
+  return atom.getConfigDirPath() ?? process.env.ATOM_HOME ?? path.join(os.homedir(), '.atom')
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e65c7060769cc2440f5823eddb7696920b92f124/types/atom/src/atom-environment.d.ts#L272

When will this function return null?

Reply:
Never unless set to null deliberately. I didn't want to delete your code 😄

Reply:
I think I remember why I didn't use that function originally now, standard node.js installs (which package-deps supports/allows) don't have Atom globals

Can you please update this to handle standard node env?

Reply:
Oh, I see. Fixed in 
https://github.com/steelbrain/package-deps/pull/335/commits/d973569b0ce91471071a0d5fbdd75ae2caca6dac